eng/common/config/ci-triggers.ts

```diff
@@ -0,0 +1,22 @@
+import { AreaPaths } from "./labels.js";
```

Comment:
Kept it here for now, if we feel like this should move in here or another file, can do that.